web/templates/settings/notification_settings.hbs

```diff
@@ -29,7 +29,14 @@
             <tbody>
                 {{#each general_settings}}
                     <tr>
-                        <td>{{ this.label }}</td>
+                        <td>
+                            {{ this.label }}
+                            {{#if (eq this.label (t "Followed topics"))}}
```

Comment:
It’s generally best to avoid performing logical operations directly within the templates.

Reply:
Hmm, I see will make this logical operation in settings_config.ts file

Reply:
Fixed